cypress/integration/userCanSetDestination.feature.js

```diff
@@ -0,0 +1,53 @@
+describe("User is shown destination input field", () => {
+  beforeEach(() => {
+    cy.server();
+    cy.visit("/");
+  });
+
+  it("can submit destination successfully", () => {
+    cy.route({
+      method: "POST",
+      url: "http://localhost:3000/api/**",
+      response: 200
+    });
+
+    cy.route({
```

Comment:
why are you stubbing out a "GET" request?

Reply:
need it apparently ...